python/pyspark/sql/tests/pandas/helper/helper_pandas_transform_with_state.py

```diff
@@ -227,6 +229,14 @@ def row(self):
         return RowMinEventTimeStatefulProcessor()
 
 
+class StatefulProcessorCompositeTypeFactory(StatefulProcessorFactory):
+    def row(self):
+        return RowStatefulProcessorCompositeType()
+
+    def pandas(self):
+        return StatefulProcessorCompositeType()
```

Comment:
nit: Please add `Pandas` as prefix of the class, and switch the method order as convention.